Description: Reverse sound overdrive changes
 Upstream has moved the raven sound-overdrive capability
 to budgie-control-center.
 This patch reverts this change until budgie-control-center
 is accepted into the archive.
Author: David Mohammed <fossfreedom@ubuntu.com>
Origin: Commit - 6a2fc9dfdbcaccc0d03f3294335607e2bcf2e7ec
Forwarded: not-needed
Reviewed-By: David Mohammed <fossfreedom@ubuntu.com>
Last-Update: 2022-02-09

--- budgie-desktop-10.5.3+gitxx.orig/src/panel/settings/settings_raven.vala
+++ budgie-desktop-10.5.3+gitxx/src/panel/settings/settings_raven.vala
@@ -15,6 +15,7 @@ namespace Budgie {
 	*/
 	public class RavenPage : Budgie.SettingsPage {
 		private Gtk.ComboBox? raven_position;
+		private Gtk.Switch? allow_volume_overdrive;
 		private Gtk.Switch? enable_week_numbers;
 		private Gtk.Switch? show_calendar_widget;
 		private Gtk.Switch? show_sound_output_widget;
@@ -61,6 +62,11 @@ namespace Budgie {
 				_("Set which side of the screen Raven will open on. If set to Automatic, Raven will open where its parent panel is.")
 			));
 
+			allow_volume_overdrive = new Gtk.Switch();
+			grid.add_row(new SettingsRow(allow_volume_overdrive,
+				_("Allow raising volume above 100%"),
+				_("Allows raising the volume via Sound Settings as well as the Sound Output Widget in Raven up to 150%.")
+			));
 			enable_week_numbers = new Gtk.Switch();
 			grid.add_row(new SettingsRow(enable_week_numbers,
 				_("Enable display of week numbers in Calendar"),
@@ -99,6 +105,7 @@ namespace Budgie {
 
 			raven_settings = new Settings("com.solus-project.budgie-raven");
 			raven_settings.bind("raven-position", raven_position, "active-id", SettingsBindFlags.DEFAULT);
+			raven_settings.bind("allow-volume-overdrive", allow_volume_overdrive, "active", SettingsBindFlags.DEFAULT);
 			raven_settings.bind("enable-week-numbers", enable_week_numbers, "active", SettingsBindFlags.DEFAULT);
 			raven_settings.bind("show-calendar-widget", show_calendar_widget, "active", SettingsBindFlags.DEFAULT);
 			raven_settings.bind("show-sound-output-widget", show_sound_output_widget, "active", SettingsBindFlags.DEFAULT);
